### ISSUE TITLE
Die shade issue

### ISSUE THREAD

**Issue by 0xtceb** [GantMan/learn-tfjs#11]
https://github.com/GantMan/learn-tfjs/blob/0c0dbc8e5404c0fe57e3f03a89141e9efd49e90a/chapter12/node/node-dice-data/src/index.js#L60

Aren't you creating a new rank 2 Tensor filled with 1 everywhere instead of copying and shading an existing dice ?

**Comment by GantMan** [comment]
Correct.  I believe this was to handle shades of gray from white to black.  However, I agree, it might be better to shade actual dice.  If you get better results, please share.  I strongly believe the dice example could be significantly improved, but I didn't want to get too advanced too fast.

**Comment by 0xtceb** [comment]
Your book was great to read and experiment with, arrived at this test I could not understand the purpose of shading X times out of X times the same values and what was the benefits for the training datas. If i got to this level of understanding you can blame your book for it ;) .

**Comment by GantMan** [comment]
Bwahahahaha I def do blame the book  <3 

Generating training data is an art form.  You should see how many cool generated datasets are out there.   I do want to come back to this dice code one day and improve it.   I feel like I 2x the quality of the results if I wanted to.  Maybe one day.
